websites/A/Anizium/presence.ts

```diff
@@ -0,0 +1,114 @@
+interface Video {
+	paused: boolean;
+	duration: number;
+	currentTime: number;
+}
+
+const presence = new Presence({
+		clientId: "1094616557763710976",
+	}),
+	strings = presence.getStrings({
+		viewHome: "general.viewHome",
+	});
+
+let video: Video;
+
+presence.on("iFrameData", (msg: Video) => {
+	video = msg;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/Z3G1Ubm.png",
+		},
+		haberAd: string =
+			document
+				.querySelector(
+					"body > div.haberler > div > div > div:nth-child(7) > div > div > div.kart-header > h2"
```

Comment:
See if you can shorten selectors like these.
For example,
```suggestion
					"div.kart-header > h2"
```